git-tar/function/handler.go

```diff
@@ -360,3 +360,11 @@ func hasDockerfileFunction(functions map[string]stack.Function) bool {
 	}
 	return false
 }
+
+func buildBranch() string {
+	branch := os.Getenv("build_branch")
```

Comment:
We can also use lookupenv istead of getenv, also as I see this function appearing in some places, we can move `build_branc` in constant and the function itself in the SDK